client/src/components/OAuthLink.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+
+import { LinkButton } from './';
+import { useGenerateLinkConfig } from '../hooks';
+
+// Component rendered when user is redirected back to site from Oauth institution site.  It initiates link immediately with
+// configs that are generated with the link token, userId and itemId from local storage.
+const OAuthLink = () => {
+  const [config, setConfig] = useState({});
+  let { userId, itemId, token } = JSON.parse(
+    localStorage.getItem('oauthConfig')
+  );
+
+  const getLinkConfig = useGenerateLinkConfig(true, userId, itemId, token);
```

Comment:
(nit) `getLinkConfig` implies this is a function, not an object. Should this be just `linkConfig`?

In general its good for variable names to reflect their purpose or type

`getFoo` -> a function to get foo
`setFoo` -> a function to set foo
`foo` -> a foo
`isFoo` -> a boolean for if something is foo
`maybeFoo` -> a foo or null